sdk/src/signer/mod.rs

```diff
@@ -66,7 +66,7 @@ pub enum SignerError {
 /// The `Signer` trait declares operations that all digital signature providers
 /// must support. It is the primary interface by which signers are specified in
 /// `Transaction` signing interfaces
-pub trait Signer {
+pub trait Signer: Debug {
```

Comment:
Not sure what's the best approach in Rust to be able to `println!(` a Trait at runtime, but this is the simplest one I got working, is it okay to keep `Debug` constraint here (or should I remove it) ?

Reply:
I used it to debug some things, wasn't sure whether it's better to leave it out or not,

but removed now (see latest PR version)